best-practices-badge.md

```diff
@@ -0,0 +1,355 @@
+You are standing in an open field of security options, west of a Badge House, with a boarded front door. There is a small website here. Opening the website reveals a small hackmd.
```

Comment:
This line should be removed, I think it is going to confuse people.

Reply:
lol k, fair